camera/EmulatedCameraDevice.h

```diff
@@ -393,11 +394,7 @@ class EmulatedCameraDevice {
              * Return:
              *  NO_ERROR on success, or an appropriate error status.
              */
-            inline status_t startThread(bool one_burst)
-            {
-                mOneBurst = one_burst;
-                return run(NULL, ANDROID_PRIORITY_URGENT_DISPLAY, 0);
```

Comment:
I think it should be ok. It affects only when camera is open/close rapidly in gUM tests. It's good on my local test. Once emulator-kk test is enable on try, I'll keep watching this part.